forloop_common_structures/database_models.py

```diff
@@ -790,17 +813,23 @@ def cast_types_to_db(self, df: pd.DataFrame) -> pd.DataFrame:
 gdtm=GenericDbTypeMapper()
 
 
-
 def escape_if_string(variable: Any) -> Any:
-    """Escape a variable if it's a string."""
+    """Escape a variable if it's a string.
+
+    Replaces single-quotes with single-quote pairs, backlashes with double-backlashes and escapes
+    double-quotes.
+
+    Note:
+        Old implementation using db connector was moved to a new function
+        'escape_if_string_database_dependent'.
+    """
     if isinstance(variable, str):
         return variable.replace("\\","\\\\").replace("'", "\\'").replace('"', '\\"')
```

Comment:
Here you had replace("'", "''") - that doesn't make sense to me, or why was it like that? I changed it to my version of this